Gruntfile.js

```diff
@@ -53,14 +52,16 @@ try {
 }
 
 
-// `TSFiles` gets a list of our TypeScript files in ./test and ./src for tslint'ing.
-// Exclude TS declaration files (*.d.ts).
-// Exclude notifications subdirectory due to legacy linting errors.
-const TSFiles = execSync('find src test -type f -name *.ts | grep -v -e "\.d\.ts$" -e "/notifications/"')
-    .toString('utf8')
-    .replace(/\s*$/, '') // remove final newline to avoid an extra (blank) element
-    .split('\n');
-// console.log(TSFiles);
+// `TSFiles` is all our TypeScript files in ./test and ./src for tslint'ing;
+// excluding notifications subdirectory due to legacy linting errors;
+// excluding TS declaration files (*.d.ts).
+const TSFiles = [
```

Comment:
Since we don't reuse this list anywhere else, I'd put it directly in tslint configuration at this point and remove comments (the glob list explains itself)